parsing/g_constr.ml4

```diff
@@ -295,14 +301,15 @@ GEXTEND Gram
       | -> [] ] ]
   ;
   instance:
-    [ [ "@{"; l = LIST1 universe_level; "}" -> Some l
+    [ [ "@{"; l = LIST0 universe_level; "}" -> Some l
       | -> None ] ]
   ;
   universe_level:
     [ [ "Set" -> GSet
       | "Prop" -> GProp
-      | "Type" -> GType None
-      | id = name -> GType (Some id)
+      | "Type" -> GType UAnonymous
+      | "_" -> GType UUnknown
```

Comment:
It's kind of strange to have anonymous not mean underscore for universes.

Reply:
I'll revert the naming